code/modules/mob/living/carbon/alien/humanoid/update_icons.dm

```diff
@@ -68,27 +68,26 @@
 		apply_overlay(HANDCUFF_LAYER)
 
 //Royals have bigger sprites, so inhand things must be handled differently.
-/mob/living/carbon/alien/humanoid/royal/update_inv_r_hand()
+/mob/living/carbon/alien/humanoid/royal/update_inv_hands()

```

Comment:
this proc is now pretty much identical to the ```/carbon``` one, so I can remove them, that also means aliens will support multi-hands too.